lib/documents.js

```diff
@@ -373,7 +428,8 @@ async function _refresh({externalIdHash, documentHash, ttl, creatorHash}) {
   const update = {
     $set: {
       'meta.updated': now,
-      'registration.expires': new Date(now + ttl)
+      'registration.expires': new Date(now + ttl),
+      'registration.minAssuranceForResolution': minAssuranceForResolution
```

Comment:
Did we decide that we always want to set this? Or do we only want to set it if it is passed in?

Reply:
Updated to check if it's passed in.